frontend/src/v5/ui/controls/scrollArea/scrollArea.styles.tsx

```diff
@@ -44,17 +44,22 @@ const ThumbHorizontal = styled.div<{ variant: ScrollbarVariant }>`
 	border-radius: 3px;
 	height: 6px;
 	background-color: ${({ variant }) => getBackgroundColor(variant)};
+	overflow-x: hidden;
 `;
-
-export const ScrollbarWrapper = styled(Scrollbars).attrs(
-	({ variant, $hidehorizontal }: { variant: ScrollbarVariant, $hidehorizontal: boolean }) => ({
+export type ScrollAreaProps = ScrollbarProps & {
```

Comment:
why the change of scrolllArea.component?

Reply:
The scrollArea component was not doing anything other than the setting a defaultvalue for `hideHorizontal` and `variant`

Reply:
I'm a bit confused because it seems to work as it currently is, but shouldn't the ternary operator in renderThumbHorizontal be the other way around?

Reply:
When I noticed that, I got pretty confused myself. However, this PR was already big enough, so I didn't want to refactor it here also risking to break other components. Plus, the golden rule states "if it works, don't touch it" lol